### ISSUE TITLE
[Accessibility] Missing language codes

### ISSUE THREAD

**Issue by jpmcb** [bottlerocket-os/project-website#109]
**Path to the page with the problem (_do not_ include domain name):**

All pages

**Describe the problem:**

```
### Definition

Page must define the language of its principal content using a valid IANA language code.

### Purpose

Assistive technologies like screen readers and other speech output technologies need to know the languages
represented by the characters in order to speak the text content correctly.

### Techniques

- Use the LANG attribute on the HTML or XHTML element to define the principal language of the web page.
- Use the IANA codes to identify the principal language (e.g. en, fr, ...).

### Compliance
WCAG Level A, Required
WCAG Success Criteria
```

https://www.w3.org/TR/WCAG21/#language-of-page

**Describe how the site should look?**

N/a - under the hood accessibility

**Comment by stockholmux** [comment]
@jpmcb Did you use a tool to come to this?

The template includes a language code. I'm wondering if it's picking up the redirect screen on the review site or something.


![Screenshot 2023-04-06 at 3 25 11 PM](https://user-images.githubusercontent.com/1152927/230496925-baeba8c9-f383-4a16-af96-f360a978cd15.png)

**Comment by jpmcb** [comment]
Yeah, I'm using the AInspector for firefox: https://ainspector.disability.illinois.edu/

And weird. I don't see the lang atribute on the `html` element:

![Screen Shot 2023-04-06 at 3 29 52 PM](https://user-images.githubusercontent.com/23109390/230497690-86c247dd-2513-4521-a169-412aeee2ba1a.png)

**Comment by jpmcb** [comment]
Ahhh, but I do see it on the documentation page: 

![Screen Shot 2023-04-06 at 3 31 58 PM](https://user-images.githubusercontent.com/23109390/230497850-1be01c2a-6ff6-40e4-b65c-c71c4d242845.png)

Not sure why it would be missing from the home page 🤔

**Comment by stockholmux** [comment]
I do know why that is happening on the homepage only. We did some overrides in Docsy template because the i18n support was pretty shoddy (it's all or nothing - really weird way of doing it).

Anyway, I think I know a solution that will meet the accessibility requirements. Excellent catch.
